squidb-processor/src/com/yahoo/squidb/processor/data/ErrorInfo.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2015, Yahoo Inc.
+ * Copyrights licensed under the Apache 2.0 License.
+ * See the accompanying LICENSE file for terms.
+ */
+package com.yahoo.squidb.processor.data;
+
+import com.yahoo.aptutils.model.DeclaredTypeName;
+
+/**
+ * Tuple class to hold logged error info, to be written by the
+ * {@link com.yahoo.squidb.processor.plugins.defaults.ErrorLoggingPlugin}.
+ */
+public final class ErrorInfo {
+
+    public final DeclaredTypeName errorClass; // The class on which to log the error
+    public final String element; // The specific element on which to log the error (or null/empty to log on the class)
+    public final String message; // The error message to log
```

Comment:
Since these members are public, maybe the line comments could instead be javadoc like on `Field.NULL`? Although I suppose there's only a very small chance anyone will be using this directly over `modelSpec.logError()`.